examples/multiple-choice/index.html

```diff
@@ -0,0 +1,55 @@
+<title>&lt;multiple-choice&gt;</title>
+
+<link rel=stylesheet href=https://devpunks.herokuapp.com/typography.css></script>
+
+<script src=/snuggsi.es></script>
+<script name=polyfill src=//cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.0.0/webcomponents-hi-ce.js></script>
+
+<!-- import works inline body -->
+<link rel=import href=/examples/header-group>
+
+<header-group>
+  <h2 slot=header>&lt;multiple-choice&gt;</h2>
+
+  <a  slot=subheader title='See more component examples' href=./>See more component examples</a>
+</header-group>
+
+<main>
+
+<multiple-choice>
+  <blockquote>{question}</blockquote>
+
+  <fieldset>
+    <legend>Answers</legend>
+
+    <ul>
+    <template name=answers>
+      <li>
+        <label>
+          <input name=answer type=radio value={#}> {self}
+        </label>
+    </template>
+  </ul>
+
+  </fieldset>
+
+  <footer>
+    <button onclick=onprevious>👈 Previous</button>
+    <button onclick=onnext>Next 👉</button>
+  </footer>
+
+</multiple-choice>
+
+</main>
+
+<script src=/examples/multiple-choice/index.es></script>
```

Comment:
... because... content negotiation. ;-) Trust me on this one. We are responding based on content type. Similar to how `Content-Encoding` strategy works. /cc @tmornini.

Am just trying it out. I actually typically prefer your strategy @brandondees